### DISCUSSION TITLE
2024-08-20 화요일

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#309] [기록보드]
2024-08-20 화요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 기술면접 준비 | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 처리 할 사항(배포버전 체크)
- [x] : 검색 동작........ 전체 수정......... 진행중 .........
- [x] : 컨트랙트 추가시 알림 동작 이상하게 작동하는 부분 확인
- [x] : 디자이너님 수정 사항 - 검색 텍스트 14pt
- [x] : 디자이너님 수정 사항 - 추천인기버디즈, 여정목록에서 새도우 약하게
- [x] : 디자이너님 수정 사항 - 여정 상세 모바일에서 스트로크로
- [x] : 목데이터 - 버디즈
- [x] : 목데이터 - 여정

### 처리 된 사항
- [x] : 여정 삭제시 에러 관련
- [x] : 소셜로그인시 인가 상태 안바뀜
- [x] : vercel analytics 적용
- [x] : 로그아웃시에도 알림 숫자 떠있음
- [x] : ai 그림 생성시 타임아웃
- [x] : 스토리 쓰기 시 필터 목록 이상함
- [x] : 스토리 없어도 작성할 수 있게 메인에 빈 카드 띄우기
- [x] : 스토리 생성 완료시 잘못된 곳으로 감
- [x] : 플필페이지 팔로잉/ 팔로우 초기값
- [x] : 비번 변경이 소셜로그인이면 안되게끔
- [x] : 팔로우 / 팔로잉 디자인
- [x] : 여정 수정페이지 전체적으로 검토
- [x] : 여정 수정페이지 한번 진입하면 헤더에 수정이 자꾸 따라옴
- [x] : 마이페이지에서 헤더 톱니바퀴 눌렀을 때 온보딩으로 감 - 이부분은 유저가 톱니바퀴를 클릭했을때 전체 수정할 수 있기를 기대할 것 같아서 유지합니다
- [x] : 알림 및 메시지 카운트 정상작동 확인
- [x] : 나가기 / 대기중 / 참여하기 동작 제대로 이루어지도록 수정
- [x] : isRead 상태에 따라서 보여지기 토글
- [x] : UI 정돈 - 여정카드, 디테일 모바일/데스크탑
- [x] : 이메일 회원가입 비밀번호 강제하기
- [x] : 탑배너 프리페치쿼리
- [x] : 스토리 관련 로직 전체 검토
- [x] : 튜토리얼 반응형 으로 수정
- [x] : 알림 기능 기본
- [x] : 알림 기능 - 참여자가 여러명일때 순차적으로 띄우기 처리
- [x] : 알림 기능 - 참여자 알림 로직 오류 없게 수정
- [x] : 오늘날짜를 여정 시작일로 할 경우 뜨는 에러 처리 - 에러가 안남 / 재현불가
- [x] : UI 정돈 전체적으로- 메인 모바일/데스크탑
- [x] : 로그아웃 해도 팔로우버튼 에러 (나가져도 팔로우 체크하는듯)
- [x] : 온도지수 계산 sql
- [x] : 스테일 타임 설정
- [x] : '실명으로 작성해 주세요' 표기
- [x] : 팔로우 버튼 스타일 일관되게 수정

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- PPT, 리드미 작성
